src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/URLValidation.feature.disabled

```
@@ -0,0 +1,385 @@
+             When a PUT request is made to "/ed-fi/schools/" with
+                  """
+                  {
+                      "schoolId": 4,
+                      "nameOfInstitution": "UT Austin College of Education Graduate",
+                      "educationOrganizationCategories": [
+                          {
+                              "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"
+                          }
+                      ],
+                      "gradeLevels": [
+                          {
+                              "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Postsecondary"
+                          }
+                      ]
+                  }
+                  """
+             Then it should respond with 405
+              And the response body is
+                  """
+                  {
+                      "detail": "The request construction was invalid.",
+                      "type": "urn:ed-fi:api:method-not-allowed",
+                      "title": "Method Not Allowed",
+                      "status": 405,
+                      "correlationId": null,
+                      "errors": [
+                         "Resource collections cannot be replaced. To \\"upsert\\" an item in the collection, use POST. To update a specific item, use PUT and include the \\"id\\" in the route."
+                      ]
+                  }
+                  """
+              And the headers contain
+                  """
+                  Content-Type: application/json; charset=utf-8
+                  """
+
+        @ignore
+        Scenario: 11 - Ensure clients cannot delete a reource if the ID of the resource is missing as part of the path
+            Given the system has the following "schools"
```

We don't require this because the request should fail due to the missing identifier

---

Removed